unl/wdntemplates
# Fix body scroll bug

Fix body scroll bug with desktop nav closing without clicking close and resizing between mobile and desktop.

To test:
* In desktop verify that when nav is open and nav closes without clicking it closed, that the body scroll is not locked.
* Verify that moving between desktop and mobile with nav open that nav and body scroll behave correctly.
*  Verify that moving between mobile and desktop with nav open that nav and body scroll behave correctly.
*  Verify that moving between desktop and mobile with nav closed that nav and body scroll behave correctly.
*  Verify that moving between mobile and desktop with nav closed that nav and body scroll behave correctly.
